lib/concurrent.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+var Promise = require('bluebird');
+var isPromise = require('is-promise');
+var BAIL_ERROR = new Error();
+
+module.exports = Concurrent;
+
+function Concurrent(tests, bail) {
+	if (!this instanceof Concurrent) {
+		throw new Error('must use `new Concurrent`');
```

Comment:
Should be `TypeError` to match ES2015 classes and message: `Class constructor Concurrent cannot be invoked without 'new'`